.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Integration Tests
+
+on:
+    - pull_request
+    - push
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the branch name
+        id: get_branch
+        run: echo ::set-output name=branch::$(echo ${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}})
+
+      - name: Get the originating repository
+        id: get_repo
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo ::set-output name=repo::${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
GitHub updated their way to output a value, and you can use something like `echo "REPOSITORIES=${repositories}" >> $GITHUB_OUTPUT` - you can also use almost the same way to share some value between jobs (and not just steps)

one example here: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#workflow

Reply:
Oh yeah, good call. I was a bit surprised seeing this syntax but assumed there was a reason. The new output stuff is *much better*.